src/main.rs

```diff
@@ -196,24 +205,49 @@ fn main() {
         tooltip += &format!("{}</b>\n", date.format(args.date_format.as_str()));
 
         if args.fahrenheit {
+            if args.nerd {
+                tooltip += &format!(
+                    "󰳡 {}° 󰳛 {}° ",
+                    day["maxtempF"].as_str().unwrap(),
+                    day["mintempF"].as_str().unwrap(),
+                );
+            } else {
+                tooltip += &format!(
+                    "⬆️ {}° ⬇️ {}° ",
+                    day["maxtempF"].as_str().unwrap(),
+                    day["mintempF"].as_str().unwrap(),
+                );
+            }
+        } else {
+            if args.nerd {
+                tooltip += &format!(
+                    "󰳡 {}° 󰳛 {}° ",
+                    day["maxtempF"].as_str().unwrap(),
+                    day["mintempF"].as_str().unwrap(),
+                );
+            } else {
+                tooltip += &format!(
+                    "⬆️ {}° ⬇️ {}° ",
+                    day["maxtempF"].as_str().unwrap(),
+                    day["mintempF"].as_str().unwrap(),
+                );
+            }
+        };
```

Comment:
Also this code has too much repetition, which made it easy to miss the fact that everything shows `fahrenheit` now.

Reply:
Oops, didn't notice, I fixed it though